config/workload_all.xml

```diff
@@ -23,6 +23,7 @@
     <hikariConnectionTimeoutMs>180000</hikariConnectionTimeoutMs>
     <useStoredProcedures>true</useStoredProcedures>
     <displayEnhancedLatencyMetrics>false</displayEnhancedLatencyMetrics>
+    <trackPerSQLStmtLatencies>true</trackPerSQLStmtLatencies>

```

Comment:
lets make it false by default?

Reply:
I think it is good to have these on by default.

 Unless we see a significant performance cost to adding this, I'd say have this on by default.